content/influxdb/v2.0/reference/cli/influx/backup/_index.md

```diff
@@ -22,20 +22,21 @@ influx backup [flags] path
-|      | `--org-id`        | Organization ID (mutually exclusive with `--org`)                          | string     | `INFLUX_ORG_ID`       |
-|      | `--skip-verify`   | Skip TLS certificate verification                                          | string     |                       |
-| `-t` | `--token`         | Authentication token                                                       | string     | `INFLUX_TOKEN`        |
+| Flag |                   | Description                                                                                                | Input type | {{< cli/mapped >}}    |
+| ---- | ----------------- | ---------------------------------------------------------------------------------------------------------- | ---------- | --------------------- |
+| `-c` | `--active-config` | CLI configuration to use for command                                                                       | string     |                       |
+|      | `--bucket-id`     | ID of the bucket to back up from (mutually exclusive with `--bucket`)                                      | string     |                       |
+| `-b` | `--bucket`        | Name of the bucket to back up from (mutually exclusive with `--bucket-id`)                                 | string     |                       |
+|      | `--compression`   | By default, `gzip` argument enables compression on downloaded files. Set to `none` to disable compression. | string     |                       |
+|      | `--configs-path`  | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`)                                      | string     | `INFLUX_CONFIGS_PATH` |
+| `-h` | `--help`          | Help for the `backup` command                                                                              |            |                       |
+|      | `--hide-headers`  | Hide table headers (default `false`)                                                                       |            | `INFLUX_HIDE_HEADERS` |
+|      | `--host`          | HTTP address of InfluxDB (default: `http://localhost:8086`)                                                | string     | `INFLUX_HOST`         |
+|      | `--json`          | Output data as JSON (default `false`)                                                                      |            | `INFLUX_OUTPUT_JSON`  |
+| `-o` | `--org`           | Organization name (mutually exclusive with `--org-id`)                                                     | string     | `INFLUX_ORG`          |
+|      | `--org-id`        | Organization ID (mutually exclusive with `--org`)                                                          | string     | `INFLUX_ORG_ID`       |
+|      | `--skip-verify`   | Skip TLS certificate verification                                                                          | string     |                       |
```

Comment:
this has a env var now

Reply:
Yes, sorry about the confusion @russorat! this update is in another PR...I'm pulling in the other PR